apps/cli/src/carize.ts

```diff
@@ -0,0 +1,62 @@
+import chalk from "chalk";
+import { filesFromPaths } from "files-from-path";
+import fs from "fs-extra";
+import type { Block } from "ipfs-car";
+import { CAREncoderStream, createDirectoryEncoderStream } from "ipfs-car";
+import path from "path";
+import { Writable } from "stream";
+
+export const createCarFromDirectory = async (
```

Comment:
This is an util function.
In an util function don't call `console.log` or do `process.exit`.
Return whatever you need to return, that is needed outside.